pkg/webui/console/views/app/app.styl

```diff
@@ -12,18 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+.container
```

Comment:
Instead of sticking the sidebar to the top, I've adjusted the layout to stay 100% always and then making the `main` scrollable instead. This makes the layout simpler and avoids weird edge cases with `sticky/fixed`.